clients/wallet/src/resubmissions.rs

```diff
@@ -26,10 +26,26 @@ const MAXIMUM_TX_FEE: u32 = 10_000_000; // 1 XLM
 
 #[cfg_attr(test, mockable)]
 impl StellarWallet {
+	/// sends a signal to stop the resubmission task
+	pub async fn stop_periodic_resubmission_of_transactions(&mut self) {
+		match &self.resubmission_end_signal {
+			None => {
+				tracing::warn!("stop_periodic_resubmission_of_transactions(): no schedule to stop");
```

Comment:
Let's maybe change the level to `debug!` as I assume this message will always be shown on the initial startup of the vault client (ie. when not triggered by a periodic restart) and it might be confusing to some.